bootstrap/config/config.go

```diff
@@ -212,8 +215,14 @@ func (cp *Processor) Process(
 
 	// Now load the private config from a local file if any of these conditions are true
 	if !useProvider || !cp.providerHasConfig || cp.overwriteConfig {
+		requestTimeout, err := time.ParseDuration(environment.GetFileUriTimeout(cp.lc))
+		if err != nil {
+			cp.lc.Warn("Failed to parse Service.RequestTimeout configuration value: %v, using default value to load config from URI", err)
```

Comment:
```suggestion
			cp.lc.Warnf("Failed to parse Service.RequestTimeout configuration value: %v, using default value %v to load config from URI", err, defaultTimeout)
```

Reply:
We agreed that this would be encapsulated in the file.Load funtion and remove the requestTimeout parameter.